docs/Using-the-metaImport-script.md

```diff
@@ -10,10 +10,11 @@ and then run the following command:
 ```
 This will tell you the parameters you can use:
 ```
-$ ./metaImport.py -h
+ $./metaImport.py  -h
```

Comment:
leading space added should be a space after `$`

Reply:
also an extra whitespace was added after `.py`

Reply:
corrected.